gammapy/detect/test_statistics.py

```diff
@@ -207,10 +330,12 @@ def _ts_value(position, counts, exposure, background, kernel, flux,
     TS : float
         TS value at the given pixel position.
     """
+    from imageutils import extract_array_2d
+
     # Get data slices
-    counts_slice = extract_array(counts, kernel.shape, position).astype(float)
-    background_slice = extract_array(background, kernel.shape, position).astype(float)
-    exposure_slice = extract_array(exposure, kernel.shape, position).astype(float)
+    counts_slice = extract_array_2d(counts, kernel.shape, position).astype(float)
+    background_slice = extract_array_2d(background, kernel.shape, position).astype(float)
+    exposure_slice = extract_array_2d(exposure, kernel.shape, position).astype(float)
```

Comment:
This change was not intended. As I had some merge conflicts, I just copied the changed files, but obviously they included some old stuff. Thanks for the hint!